pkgs/by-name/ju/jumper/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "v0"; # what should I put here? No version found anywhere
+  src = fetchFromGitHub {
+    owner = "homerours";
+    repo = "jumper";
+    rev = "264f291";
+    sha256 = "sha256-6++r0mBrV+KhQc7WRiKd8l52CKJUcvzPFqe4S6O6J84=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.which # removes comp warning
+  ];
+
+  buildInputs = [
+    pkgs.fzf # enables fuzzy finding
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp jumper $out/bin
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Jumper helps you jumping into directories and files that you frequently visit";
+    homepage = "https://github.com/homerours/jumper";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ camerondugan ];
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ camerondugan ];
    mainProgram = "jumper";
    platforms = lib.platforms.unix; # not sure. please change this to the actual list of platforms jumper supports and builds on.
```

Reply:
I'm not sure either, It definitely builds on x86-linux

Reply:
Leave it with `unix` and we'll wait and see what OfBorg builds. The macOS builds may take a while though.

Reply:
Builds on aarch64-linux. https://github.com/NixOS/nixpkgs/pull/337490/checks?check_run_id=29267246551